math/src/main/java/com/powsybl/math/graph/UndirectedGraphImpl.java

```diff
@@ -115,6 +115,9 @@ private void checkEdge(int e) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
True for all other methods : inheriting doc is the default behaviour, so this is useful only when you want to add some javadoc.